spec/lib/cms/audit/pages_spec.rb

```diff
@@ -87,6 +87,10 @@
         ]
       end
 
+      it 'used UTF-8 encoding' do
+        expect(described_class::ENCODING).to eq 'UTF-8'
+      end
```

Comment:
This is one of the few times you will see me saying "you don't need a test here" :laughing: 
Basically we're testing a constant value, not business logic, so becomes a little bit redundant IMHO. 
But not a big issue!